sencha-workspace/packages/slate-cbl/sass/etc/all.scss

```diff
@@ -7,3 +7,8 @@
     $level-colors-medium,
     $level-colors-dark
 );
+
+div[id^='slate-cbl-'][id*='-picker'],
+.x-css-shadow {
+    z-index: 10 !important;
+}
```

Comment:
It's not a good idea to monkey-patch the generated ExtJS styling like this, and the selector is liable to not catch everything ExtJS might put above the omnibar

A better approach instead of this would be to find the existing place where `.slate-omnibar` is assigned a `z-index: 99` and set that to `z-index: 999999` as our intention is that the omnibar should cover EVERYTHING. That way we don't have to get into whack-a-moleing specific things to be lower than it